include/cuco/operator.hpp

```diff
@@ -50,6 +50,12 @@ struct erase_tag {
 struct contains_tag {
 } inline constexpr contains;  ///< `cuco::contains` operator
 
+/**
+ * @brief `count` operator tag
+ */
+struct count_tag {
```

Comment:
Nit: Should we rename this to `count/find/*_operator` in a future PR?

Reply:
Are you suggesting to replace `*_tag` with `*_operator`?

Reply:
Yeah, I find my intial choice with "_tag" might not be ideal

Reply:
If course that's just a general remark - no action item for this PR

Reply:
It's like the `empty_value` v.s. `empty_payload` discussion we just had. `*_tag` looks fine to me.